File: Treasury/Models/Financial Models/Funding Request Models/FundingRequest.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Treasury.Models.Financial_Models.Funding_Request_Models
{
    [Table("Complete Funding Request")]
    public class FundingRequest
    {
        [Column("ID")] 
        public int ID { get; set; }
        
        [Column("Name of Club")]
        public string NameOfClub { get; set; }

        [Column("Hearing Date")]
        public DateTime HearingDate { get; set; }
        
        [Column("Dot Number")] 
        public string DotNumber { get; set; }
        
        [Column("Description")] 
        public string Description { get; set; }

        [Column("Fiscal Year")]
        public string FiscalYear { get; set; }

        [Column("Requested")]
        public decimal AmountRequested { get; set; }

        public string Decision { get; set; }

        [Column("Approved")]
        public decimal AmountApproved { get; set; }
    }
}

File: src/main/java/edu/wpi/sga/treasury/domain/model/FundingRequest.java
package edu.wpi.sga.treasury.domain.model;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import java.math.BigDecimal;
import java.time.Instant;
import java.time.LocalDate;

@Getter
@Setter
@Entity
@Table(name = "`Funding Requests`")
public class FundingRequest {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "ID", nullable = false)
    private Integer id;

    @ManyToOne(optional = false)
    @JoinColumn(name = "`Name of Club`", nullable = false)
    private Organization organization;

    @Column(name = "Description")
    private String description;

    @Column(name = "`Funding Date`", nullable = false)
    private LocalDate fundingDate;

    @Column(name = "`Fiscal Year`", length = 6)
    private String fiscalYear;

    @Column(name = "`Date of Event`")
    private LocalDate dateOfEvent;

    @Column(name = "`Dot Number`", nullable = false, length = 6)
    private String dotNumber;

    @Column(name = "`Amount Requested`", nullable = false, precision = 10, scale = 2)
    private BigDecimal amountRequested;

    @Column(name = "Decision", nullable = false, length = 20)
    private String decision;

    @Column(name = "`Amount Approved`", nullable = false, precision = 10, scale = 2)
    private BigDecimal amountApproved;

    @Column(name = "Notes", length = 512)
    private String notes;

    @Column(name = "Timestamp", nullable = false)
    private Instant timestamp;

}